packages/core/src/actors.ts

```diff
@@ -404,9 +405,11 @@ export function fromEventObservable<T extends EventObject>(
           self.send({ type: completeEventType });
         }
       });
+
       return state;
     },
-    getSnapshot: (_) => undefined
+    getSnapshot: (_) => undefined,
+    getStatus: (state) => state
```

Comment:
This is likely a bug fix for `fromEventObservable`. I need to write a test for this case